src/worker.js

```diff
@@ -133,6 +133,7 @@ class Worker {
     return Promise.fromCallback((cb) => {
       cls.run(() => {
         cls.set('tid', this.tid)
+        cls.set('previousEvent', this.queue)
```

Comment:
Yeah, that one is better. I didn't like `event` part, because event can be published from task. So `WorkerName` is better than event. `current` also makes sense